application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -313,9 +313,12 @@ Analytics.Code.ConfigurationClass_siteId=Site ID
 Analytics.Code.ConfigurationClass_trackingCode.hint=Tracking Code received from Matomo. The 'u' variable might be different than the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
 Analytics.Code.ConfigurationClass_trackingCode=Tracking code
 admin.analytics.application=Analytics
+Analytics.Code.ConfigurationClass_enabled=Enabled
+Analytics.Code.ConfigurationClass_enabled.hint=Enable the ability to track the instance. Disabled by default.
```

Comment:
Maybe `Allow Matomo to track your instance. Disabled by default`. WDYT?

Reply:
Indeed, your suggestion is better. I will update the translation.